File: Backend/WebAPI/Requests/OrderRequest.cs
using System;
using System.Collections.Generic;

namespace WebAPI.Requests
{
    public class OrderRequest
    {
        public int SessionId { get; set; }
        public DateTime RegistratedAt { get; set; }
        public SessionSeatsRequest SessionSeats { get; set; } = null!;
        public IReadOnlyCollection<CinemaServiceRequest> CinemaServices { get; set; } = null!; 
    }
}

File: Frontend/src/app/Requests/OrderRequest.ts
import { CinemaServiceRequest } from "./CinemaServiceRequest";
import { SessionSeatsRequest } from "./SessionSeatsRequest";

export class OrderRequest {

    sessionId : number;
    registratedAt : Date;
    sessionSeats : SessionSeatsRequest;
    cinemaServices : CinemaServiceRequest[];

    constructor(
        sessionId : number,
        registratedAt : Date,
        sessionSeats : SessionSeatsRequest,
        cinemaServices : CinemaServiceRequest[]
    ) {
        this.sessionId = sessionId;
        this.registratedAt = registratedAt;
        this.sessionSeats = sessionSeats;
        this.cinemaServices = cinemaServices;
    }

}